Makefile

```diff
@@ -10,7 +10,25 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+SHELL = /usr/bin/env bash -o pipefail
+
 .PHONY: test
 test:
 	./tests/e2e.sh
 
+.PHONY: generate-bundles
+generate-bundles:
+	kustomize build ./config/examples/kube-prometheus | docker run --rm -i ryane/kfilt -i kind=CustomResourceDefinition > ./config/examples/kube-prometheus/bundle_crd.yaml
+	kustomize build ./config/examples/kube-prometheus | docker run --rm -i ryane/kfilt -x kind=CustomResourceDefinition > ./config/examples/kube-prometheus/bundle.yaml
+
+
+.PHONY: compare-bundles
+compare-bundles:
+	kustomize --load-restrictor LoadRestrictionsNone build config/examples/kube-prometheus | docker run --rm -i ryane/kfilt -i kind=CustomResourceDefinition > bundle_crd.yaml
+	kustomize --load-restrictor LoadRestrictionsNone build config/examples/kube-prometheus | docker run --rm -i ryane/kfilt -x kind=CustomResourceDefinition > bundle.yaml
```

Comment:
What about depending on the generate-bundles target, then do a `git diff --exit-code` or similar command 
https://git-scm.com/docs/git-diff